### DISCUSSION TITLE
Project page UI suggestions

### DISCUSSION THREAD

**Discussion by johardi** [redivis/meta#171] [Feature requests]
These are just my two cents:
1. Use meaningful icons for the "private vs shared" project (a good example is the Google Docs)
2. The share dialog should contain only options for sharing the project, not updating the project information
3. The "move to a new study" label should contain only the command for moving the project, not sharing the project
4. Use meaningful icons for the "full dataset vs 1% sample". A full and partial pie chart perhaps? or "100% and 1%" icon (?)
5. Use the align center or left for the project tree. If I scroll the project tree until it hits the left end, the tree goes too much to the right
6. Add a table header in the "Dataset variable" view
7. Have a visual cue in the table header for resizing the column width

**Comment by imathews** [comment]
@johardi We've incorporated many of these recommendations (and a lot more) in our [latest updates to the project tool](https://github.com/redivis/meta/releases/tag/0.20.0). More still to come with improvements to viewing data in the table.
